src/terra-dev-site/test/modal-manager/DisclosureComponent.jsx

```diff
@@ -177,6 +182,7 @@ class DisclosureComponent extends React.Component {
         {disclosureManager && disclosureManager.goBack ? <button type="button" className="go-back" onClick={this.goBack}>Go Back</button> : null}
         {disclosureManager && disclosureManager.maximize ? <button type="button" className="maximize" onClick={this.maximize}>Maximize</button> : null}
         {disclosureManager && disclosureManager.minimize ? <button type="button" className="minimize" onClick={this.minimize}>Minimize</button> : null}
+        <button type="button" className="global-close-disclosure" onClick={closeMostRecentDisclosure}>Global Close</button>
```

Comment:
it might be nice to use `getActiveDisclosureCount` in this test to show the current number of disclosures that are open

Reply:
Agreed, will update.

Reply:
Update: a8d53bc2450685c620c74f2468cc89a0c6ea11df